Configurations/release-move-tag.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+# Convenience script to automatically commit Package.swift after updating the checksum and move the latest tag
+latest_git_tag=$(git describe --abbrev=0) # gets the latest tag name
+commits_since_tag=$(git rev-list ${latest_git_tag}.. --count)
+if [ "$commits_since_tag" -gt 0 ]; then
+    # If there have been commits since the latest tag, it's highly likely that we did not intend to do a full release
+    echo "WARNING: $commits_since_tag commit(s) since tag '$latest_git_tag'. Did you tag a new version?"
+    echo "Package.swift has not been committed and tag has not been moved."
+else
+    # TODO: add sanity check to see if version is actually being updated or not?
```

Comment:
Since the script does not even proceed if there have been commits since the latest tag (i.e., we have not tagged the latest changes), I don't think it's actually necessary to check that the version number has changed. That check would actually be a little tricky.